### ISSUE TITLE
'Test Set Generation' table is missing the sar10g and u10g columns

### ISSUE THREAD

**Issue by markdouglasphd** [ITISFoundation/publication-IEC62209#19]
The table in 'Test Set Generation' should look just like the table in 'Training Set Generation'.
Please add the sar10g and u10g columns.

**Comment by SCA-ZMT** [comment]
see https://github.com/ITISFoundation/IEC62209-service/issues/77
will be fixed with https://github.com/ITISFoundation/IEC62209-service/pull/78
